enterprise-suite/README.md

```diff
@@ -89,17 +83,19 @@ Install [yq](https://github.com/mikefarah/yq) if you don't have it yet:
     # or
     brew install yq                  
 
+Make sure the image versions in `enterprise-suite/values.yaml` are as desired.
+
 Then run the release script on a clean master checkout:
 
-    scripts/make-release.sh enterprise-suite
+    ./scripts/make-release.sh enterprise-suite
     git push --follow-tags
     
 This will increment the chart version, package it, and make a
-commit. Finally, `git push --tags` will publish the release and git tag.
+commit. Finally, `git push --follow-tags` will publish the release and git tag.
```

Comment:
This isn't true anymore, it publishes before the version is incremented. Should make it clear here.

Reply:
Good point.  I fixed by tagging before making the change to the version in Chart.yaml.